### PR TITLE
Fix target current page left menu

### PR DESCRIPTION
**Pull request by aloupfor**

### CODE REVIEW

**Comment by itkg-alavieille** [comment]
:+1:

**Comment by ggrasset** [comment]
:+1:

File: BackofficeBundle/Resources/public/less/smartadminPatches/nav.less [hunk 1]
```diff
@@ -0,0 +1,24 @@
+nav {
+
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/less/smartadminPatches/nav.less]: Could you suppress the empty lines ?

Comment (aloupfor) [inline BackofficeBundle/Resources/public/less/smartadminPatches/nav.less]: https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1278

**Pull request merged**